### PR TITLE
[GOAL2-624] Add a kmd_config.json.example to the installer dir

### PR DESCRIPTION
**Pull request by EvanJRichard**
## Summary

A common request from developers is for some kind of `kmd_config.json.example` to look at and work from. This PR proposes to add such an `example` to the `installer` dir, next to the normal algod `config.json.example`. It is equivalent to the current default kmd configuration.

## Test Plan

Automated test suite will check for regressions.

### CODE REVIEW

**Comment by EvanJRichard** [comment]
_@tsachiherman wrote:_
> I think that you'll need to do bit more than just that -
> 
> 1. Format the json so it would be human readable ( unless there is a good reason not to do so )
> 2. Add this json to the following locations -
>    
>    * installer/rpm/algorand.spec
>    * scripts/build_deb.sh
>    * scripts/build_package.sh

:+1: Formatted the file for 1.), good thinking.
Re 2.) - isn't that only needed if the file is to be installed somewhere? My sole goal here is to provide an example for devs to look at.
Context: To be used, a `kmd_config.json` needs to be in the `kmd-{$version_changes_over_time}` data directory, which doesn't exist until kmd is run for the first time. So, I'm not sure it can easily be copied to the right destination from the installer scripts, the desired destination isn't yet created and its version is unknown. (I will find a way if we think it should really be installed, though! 🙂)

**Comment by tsachiherman** [comment]
> _@tsachiherman wrote:_
> 
> > I think that you'll need to do bit more than just that -
> > 
> > 1. Format the json so it would be human readable ( unless there is a good reason not to do so )
> > 2. Add this json to the following locations -
> >    
> >    * installer/rpm/algorand.spec
> >    * scripts/build_deb.sh
> >    * scripts/build_package.sh
> 
> 👍 Formatted the file for 1.), good thinking.
> Re 2.) - isn't that only needed if the file is to be installed somewhere? My sole goal here is to provide an example for devs to look at.
> Context: To be used, a `kmd_config.json` needs to be in the `kmd-{$version_changes_over_time}` data directory, which doesn't exist until kmd is run for the first time. So, I'm not sure it can easily be copied to the right destination from the installer scripts, the desired destination isn't yet created and its version is unknown. (I will find a way if we think it should really be installed, though! 🙂)

The formatting looks weird; I tried to format it myself and came up with this  https://github.com/tsachiherman/go-algorand/tree/tsachi/goal2-624
to figure this out, I used this web site - https://jsonformatter.org

As for the rest - I'm not particularly picky. If you think that it would only be used by people who are locally compiling our source code, then I'm fine with it.

File: daemon/kmd/config/config.go [hunk 1]
```diff
@@ -95,8 +96,15 @@ func LoadKMDConfig(dataDir string) (cfg KMDConfig, err error) {
 	cfg = defaultConfig(dataDir)
 	configFilename := filepath.Join(dataDir, kmdConfigFilename)
 	dat, err := ioutil.ReadFile(configFilename)
-	// Return the default configuration if the file doesn't exist
+	// If there is no config file, then return the default configuration, and dump the default config to disk
 	if err != nil {
+		exampleFilename := filepath.Join(dataDir, kmdConfigExampleFilename)
```
Comment (Karmastic) [inline daemon/kmd/config/config.go]: Use codes.SaveObjectToFile() and save it formatted.

Comment (bylun) [reply daemon/kmd/config/config.go:103]: Where to modify the default configuration ip and port?

**Pull request merged**